visualization/app/codeCharta/ui/colorPickerForMapColor/mapColorLabel.pipe.ts

```diff
@@ -5,14 +5,35 @@ import { MetricMinMax } from "../../state/selectors/accumulatedData/metricData/s
 
 @Pipe({ name: "mapColorLabel" })
 export class MapColorLabelPipe implements PipeTransform {
-	transform(metricName: keyof MapColors, colorRange: ColorRange, nodeMetricRange: MetricMinMax): string {
+	transform(metricName: keyof MapColors, colorRange: ColorRange, nodeMetricRange: MetricMinMax, colorMetric: string): string {
 		switch (metricName) {
-			case "positive":
-				return `${nodeMetricRange.minValue} to < ${this.formatNumber(colorRange.from)}`
-			case "neutral":
-				return `≥ ${this.formatNumber(colorRange.from)} to ≤ ${this.formatNumber(colorRange.to)}`
-			case "negative":
-				return `> ${this.formatNumber(colorRange.to)} to ${this.formatNumber(nodeMetricRange.maxValue)}`
+			case "positive": {
+				const isColorMetricUnary = colorMetric === "unary"
+				const isFromValueEqualMinValue = nodeMetricRange.minValue === colorRange.from
+				const isFromValueEqualMaxValue = nodeMetricRange.maxValue === colorRange.from
+				return isColorMetricUnary
+					? `${nodeMetricRange.minValue} - ${nodeMetricRange.minValue}`
```

Comment:
The minimum and maximum value of the unary metric both get set to 1 in the code (somewhat arbitrarily). For this pipe to not break in the future, I'll change the code to use the minimum and maximum values of the unary instead of the minimum value only.